mysql-test/main/backup_locks.test

```diff
@@ -188,6 +188,79 @@ DROP TABLE t3;
 BACKUP UNLOCK;
 DROP TABLE t3;
 
+--echo #
+--echo #  MDEV-28367: BACKUP LOCKS on table to be accessible to those
+--echo #              with database LOCK TABLES privileges
+--echo #
+
+--source include/have_metadata_lock_info.inc
+create database db1;
+create table db1.t1(t int);
+create user user1@localhost;
+select user,host from mysql.user where user='user1';
+--error ER_DBACCESS_DENIED_ERROR
+--connect (con1, localhost, user1, ,db1)
+
+# Global privileges should be granted to all schemas, not individual DB
+--error ER_WRONG_USAGE
+grant reload on db1.* to user1@localhost;
+grant reload on *.* to user1@localhost;
+show grants for user1@localhost;
+--error ER_DBACCESS_DENIED_ERROR
+--connect (con1, localhost, user1, ,db1)
```

Comment:
This also isn't testing the functionality you've added.

Reply:
As above comment for line `206`.

Reply:
Same as for line 206.

Reply:
like previous moved test result. Put grant tests in `grant.test`.